Dockerfile

```diff
@@ -27,6 +27,16 @@ RUN set ex \
     && rm -rf /var/lib/apt/lists/* \
     && chmod +x /usr/bin/tini
 
-COPY . .
+# Group 'python' (GID 999) and user 'python' (uid 999) are created
+RUN groupadd -g 999 python && useradd -u 999 -r -g python python
+
+# App folder and '/nginx' mount point are created with the new user as owner
+RUN mkdir -p /nginx && chown -R python:python /nginx .
```

Comment:
You can colapse these two `RUN` into one to reduce the number of layers produced by the image.

```suggestion
# Group 'python' (GID 999) and user 'python' (uid 999) are created
RUN groupadd -g 999 python && \
    useradd -u 999 -r -g python python && \
    # App folder and '/nginx' mount point are created with the new user as owner
    mkdir -p /nginx && chown -R python:python /nginx .
```